.github/scripts/verify_changes.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+event_type=$1
+base_ref=$2
+
+changed_dir=""
+
+git fetch --no-tags $base_ref
+
+if [[ "$1" == "pull_request" ]]; then
+  change_count=$(git diff --merge-base $base_ref --name-only |awk -F"/" '{ print $1}' |uniq |wc -l)
```

Comment:
formatting nit, would make it a little clearer to follow with a space after the pipes as well!

Reply:
Fixed 👍